### DISCUSSION TITLE
Set content-type for parameters in multipart/form-data

### DISCUSSION THREAD

**Discussion by shipko** [hoppscotch/hoppscotch#4032] [Ideas]
Hello everyone 
With this discussion, I want to start a discussion on how to make a request in Hoppscotch similar to Postman.
Judging by the browser documentation, we cannot set a separate parameter type, we may need to do some kind of hack.
<img width="1287" alt="Снимок экрана 2024-05-03 в 15 13 39" src="https://github.com/hoppscotch/hoppscotch/assets/1519243/bba6c50c-9441-4c7a-972f-2d677982d766">

**Comment by SanskritiHarmukh** [comment]
Hi @shipko , Can you describe specific use cases where you find it necessary to set content types for individual parts of a multipart/form-data request?

**Reply by shipko** [reply]
Hi, @SanskritiHarmukh!
Sure, in most of enterprise software this approach is  frequently encountered not in browser, but in API with server-server communication.
I can't to take examples because NDA, but it's a critical part for transferring to Hoppscotch

**Reply by SanskritiHarmukh** [reply]
Hi @shipko, we have rolled out the feature to set content types for specific parameters in `multipart/form-data`. You can find the detailed instructions on how to set it up in our [documentation](https://docs.hoppscotch.io/documentation/getting-started/rest/uploading-data#set-content-type-for-specific-parameters-in-multipart-form-data).
